CONTRIBUTING.md

```diff
@@ -1,63 +1,28 @@
 How to report a bug
 -------------------
-- Use the search function to see if someone else has already reported the bug. If an issue has already been opened, see if you can add any useful information to it.
-- Otherwise, create a new issue on the "issues" page
-- Describe the problem as detailed as possible.
-- Some bugs only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`) as well as which version of AntennaPod you are using.
-- If the bug only seems to occur with a certain podcast, make sure to include the URL of that podcast.
-- If possible, add instructions on how to reproduce the bug.
-- If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with an app like `aLogcat`. Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it.
-- Usually, you can make a screenshot of your smartphone by pressing [Power] + [Volume down] for a few seconds
-- Please use the following **template**: 
-
-**App version**: 1.x (from Google Play/F-Store/Custom build)
-
-**Android version**: 5.x [Especially mention custom roms!]
-
-**Device model**: 
-
-**Expected behaviour**: 
-
-**Current behaviour**: 
-
-**First occured**: Version 1.x / about x days/weeks ago
-
-**Steps to reproduce**:
-
-1. Do this
-1. Do that
-
-**Environment**: Settings you have changed, e.g. Auto Download. "Unusual" devices you use, e.g. Bluetooth headphones.
-
-**Stacktrace/Logcat**: [if available]
-```
-...
-```
+- See the [template](.github/ISSUE_TEMPLATE/bug_report.md). Read the DELETE ME text when submitting.
 
```

Comment:
Please only delete the part about the template. Everything else is useful, I think.